### ISSUE TITLE
Is there a risk of exposing my mac address by using this script?

### ISSUE THREAD

**Issue by Fluorite8** [flashnuke/wifi-deauth#17]

**Comment by flashnuke** [comment]
the script explicitly sets the access point's mac address as the "source" mac address when sending spoofed packets, see the `ap_mac` and how it's used when crafting the `RadioTap()` layers:
https://github.com/flashnuke/wifi-deauth/blob/main/wifi_deauth/wifi_deauth.py#L284
https://github.com/flashnuke/wifi-deauth/blob/main/wifi_deauth/wifi_deauth.py#L297
https://github.com/flashnuke/wifi-deauth/blob/main/wifi_deauth/wifi_deauth.py#L307

in other words what this means is that instead of your mac address, these packets contain the target access point's mac address, so there should be no risk of exposing yours

however if that's important to you, you can use a 3rd party tool such as `macchanger` to spoof your device's mac address, just be sure you use it on the right network interface
